src/tile/tile.ts

```diff
@@ -14,21 +14,23 @@ export type Color = (typeof colors)[number];
 export type Symbol = (typeof symbols)[number];
 
 export type Tile = PlayableTile | NeutralTile;
-
+type Styles = "unallowed" | "allowed";
```

Comment:
The name 'Styles' implies a general styling capability, where you actually only support allowed or unallowed. Why not simply name the field `allowedAsFirstMove` (or `allowedAsNextMove` if you plan to reuse this logic for the 2+ turns as well) and set the type to a simple `boolean`?

Reply:
True ! I replaced it with "moveAllowed" property.